modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk.clj

```diff
@@ -411,7 +414,7 @@
     (log/infof (trs "DB {0} had hardcoded dataset-id; changing to an inclusion pattern and updating table schemas"
                     db-id))
     (try
-      (db/execute! {:update MetabaseTable
+      (db/execute! {:update (t2/table-name MetabaseTable)
```

Comment:
A few people were asking for this after I was using some of the table names directly in the previous PRs, so I added it in to Toucan 2